src/main/java/pages/WebDriverInit.java

```diff
@@ -23,6 +24,16 @@ public static WebDriver getDriver() {
         return driver;
     }
 
+    public static WebDriver getFirefoxDriver() {

```

Comment:
please, use factory method here and ThreadLocal class to ensure thread safety. As an example, your Driver class may look like:
```
public class Driver {

    // 'mvn clean -Dbrowser=firefox test ' command line contains a parameter  for determining the browser type. The 
    // same for remote:  '-Dremote=http://test:test-password@GRID-HOST:4444/wd/hub'
    private static final String browser = System.getProperty("browser", "chrome");
    private static final String remote = System.getProperty("remote");

    private static final ThreadLocal<WebDriver> driver = new ThreadLocal<>();

    private Driver() {
    }

    public static WebDriver getDriver() {
        if (driver.get() == null) {
            driver.set(createDriver());
        }
        return driver.get();
    }

    private static WebDriver createDriver() {
        MutableCapabilities capabilities = BrowserCapabilities.getBrowserCapabilities(browser);
        if (!StringUtils.isNullOrEmpty(remote)) {
            return getRemoteDriver();
        }
        return getLocalDriver(capabilities);
    }
    
    private static WebDriver getRemoteDriver() {
        try {
            return new RemoteWebDriver(new URL(remote), BrowserCapabilities.getCloudCapabilities(browser, remote));
        } catch (MalformedURLException e) {
            throw new IllegalArgumentException("Invalid 'remote' parameter: " + remote, e);
        }
    }

    private static WebDriver getLocalDriver(Capabilities capabilities) {
        switch (browser) {
            case "chrome" -> {
                WebDriverManager.getInstance(ChromeDriver.class).setup();
                return new ChromeDriver((ChromeOptions) Objects.requireNonNull(capabilities));
            }
            case "firefox" -> {
                WebDriverManager.getInstance(FirefoxDriver.class).setup();
                return new FirefoxDriver((FirefoxOptions) Objects.requireNonNull(capabilities));
            }
            default -> fail("Please check your browser name");
        }
        return null;
    }

    public static void tearDown() {
        if (driver.get() != null) {
            driver.get().quit();
            driver.remove();
        }
    }
}
```
where class for the capabilities may look like:
```
public class BrowserCapabilities {

    public static MutableCapabilities getBrowserCapabilities(String browser) {
        switch (browser) {
            case "chrome":
                return getChromeCapabilities();
            case "firefox":
                return getFirefoxCapabilities();
            default:
                fail("getBrowserCapabilities: Please check your browser name");
        }
        return null;
    }

    private static MutableCapabilities getFirefoxCapabilities() {
        return new FirefoxOptions();
    }

    private static MutableCapabilities getChromeCapabilities() {
        ChromeOptions options = new ChromeOptions();
        options.addArguments("enable-automation");
        options.setPageLoadStrategy(PageLoadStrategy.NORMAL);
        return options;
    }
}
```